assets/js/blocks/products/all-products/save.js

```diff
@@ -15,16 +15,12 @@ export default function save( { attributes } ) {
 		.forEach( ( key ) => {
 			dataAttributes[ key ] = attributes[ key ];
 		} );
```

Comment:
Pretty sure line 12-17 is no longer needed right?

Reply:
removed line 12-17.